recipe/meta.yaml

```diff
@@ -10,23 +11,30 @@ source:
   sha256: ce35865411f0490368a8fc383f29071de6690cbadc27704734978221f25e2bed
 
 build:
-  number: 3
   skip: true  # [win and vc!=14]
```

Comment:
I think the restriction on VS2015 only can be removed.
There used to be a run requirement introduced here:
https://github.com/AnacondaRecipes/ninja-feedstock/commit/5194af92468b8baf804da715ecc7830df6972f8d
But this was removed there:
https://github.com/AnacondaRecipes/ninja-feedstock/commit/e0489d9cb6e68e996dad532e0b7694ea6a012d9b